tock/hours/views.py

```diff
@@ -613,6 +616,71 @@ def get_object(self, queryset=None):
             user_id=self.request.user.id)
         return obj
 
+    def clean_task_data(self, float_people_id):
+        # Get Float task data for week corresponding w/ ReportingPeriod
+        # instance.
+        r = get_float_data(
+            endpoint='tasks',
+            params={
+                'weeks': base.FLOAT_API_TASK_WEEKS,
+                'start_day': datetime.datetime.strptime(
+                    self.kwargs['reporting_period'], "%Y-%m-%d"
+                ).date()
+            }
+        )
+        if r.status_code != 200:
+            return None
```

Comment:
Yeah, I hear ya. I had a whole elaborate (read: overbuilt) error reporting system in an [earlier commit](https://github.com/18F/tock/blob/1715b52925227b863d60637d28cf1059639ec62e/tock/hours/views.py#L345). A little logging might be good though, will add in a `print()` statement if response is not 200.